packages/services/src/view-service/asset-metadata-by-id.test.ts

```
@@ -55,15 +51,15 @@ describe('AssetMetadataById request handler', () => {
   });
 
   test('should successfully respond with metadata when idb record is present', async () => {
-    mockIndexedDb.getAssetsMetadata?.mockResolvedValue(metadataFromIdb);
+    mockIndexedDb.getAssetsMetadata.mockResolvedValue(metadataFromIdb);
```

optional access could silently fail, and mislead a developer into thinking they were configuring a method that was not actually mocked. so now, access is guaranteed and it's not possible to 'optionally' configure a mock.